source/shared/cpp/ObjectModel/AdaptiveCard.cpp

```diff
@@ -103,9 +103,14 @@ std::shared_ptr<AdaptiveCard> AdaptiveCard::Deserialize(const Json::Value& json)
     return result;
 }
 
-std::shared_ptr<AdaptiveCard> AdaptiveCard::DeserializeFromString(const std::string& jsonString)
+std::shared_ptr<AdaptiveCard> AdaptiveCard::DeserializeFromString(const std::string& jsonString) throw(AdaptiveCards::AdaptiveCardParseException)
 {
-    Json::Value jsonValue(jsonString);
+    Json::Reader reader;
+    Json::Value jsonValue;
+    if (!reader.parse(jsonString.c_str(), jsonValue))
```

Comment:
Thanks for fixing this. I will also send a PR for the FromString for the rest of the card elements.